addOns/reports/src/test/java/org/zaproxy/addon/reports/ExtensionReportsUnitTest.java

```diff
@@ -1149,6 +1173,53 @@ void shouldGenerateValidXmlPlusReport() throws Exception {
         assertThat(root.getAttribute("generated").length(), is(greaterThan(20)));
     }
 
+    @ParameterizedTest
+    @ValueSource(
+            strings = {
+                "modern",
+                "high-level-report",
+                "risk-confidence-html",
+                "traditional-html",
+                "traditional-html-plus",
+                "traditional-json",
+                "traditional-json-plus",
+                "traditional-md",
+                "traditional-pdf",
+                "traditional-xml",
+                "traditional-xml-plus"
+            })
+    void shouldGenerateReportsWithoutWarnings(String reportName) throws Exception {
+        // Given
+        ExtensionReports extRep = new ExtensionReports();
+        ReportData reportData = getTestReportData();
+        AlertNode root = new AlertNode(0, "Alerts");
+        root.add(newAlertNode(1, Alert.RISK_HIGH, "Alert High 1", "https://www.example.com", 1));
+        root.add(newAlertNode(2, Alert.RISK_LOW, "Alert Low 1", "https://www.example.com", 3));
+        root.add(
+                newAlertNode(
+                        2,
+                        ExtensionReports.RISK_FALSE_POSITIVE,
+                        "Alert Low 2",
+                        "https://www.example.com",
+                        8));
+        reportData.setAlertTreeRootNode(root);
+        String site1 = "https://www.example.com";
+        reportData.setSites(List.of(site1));
+        reportData.setIncludeAllConfidences(true);
+        reportData.setIncludeAllRisks(true);
+        reportData.setDescription("");
+        reportData.addSection("alerts");
+        reportData.addSection("appendix");
+        File f = File.createTempFile("zap.reports.test", "x");
+        Template template = getTemplateFromYamlFile(reportName);
+        // When
+        File r = extRep.generateReport(reportData, template, f.getAbsolutePath(), false);
+        String report = new String(Files.readAllBytes(r.toPath()));
+        // Then
+        assertThat(report.length(), greaterThan(0));
+        assertThat(logEvents, not(hasItem(startsWith("WARN"))));
```

Comment:
I guess I could also do "ERROR" here, it might take some more mocking i18N and report data 🤷‍♂️